gpkit/solution_array.py

```diff
@@ -231,14 +231,18 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
             lines.append([varstr, valstr, units, label])
         else:
             varstr = "$%s$" % varstr.replace(" : ", "")
+            boxunit = var.unitstr()
+            if not boxunit:
+                boxunit = "-"
```

Comment:
I don't think this change is necessary -- or at least it should be a user option what to print in the `dimless` case.